force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +47,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        // Check if the age is null.  If so, return null.  Otherwise, compare the age to 18 and return true if the age is 18 or older, otherwise return
+        if(age >= 18){
```

Comment:
With boolean expressions you can shorthand this by returning just the expression itself:
`return age >= 18;`